multimedia/Sunshine/Portfile

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem               1.0
+PortGroup                cmake 1.1
+PortGroup                github 1.0
+PortGroup                boost 1.0
+
+# bump revision when changes are made to this file
+revision                 1
+
+github.setup             sunshinestream sunshine master
+name                     Sunshine
+version                  0.14.0
+categories               multimedia emulators games
+platforms                darwin
+license                  GPL-3
+maintainers              {@SunshineStream sunshinestream}
+description              Sunshine is a Gamestream host for Moonlight.
+long_description         {*}${description}
```

Comment:
This description is inadequate. Unless someone knows what a "Gamestream host for Moonlight" is, they will have no idea what this port does. (I have no idea.) The Long Description should be sufficient for a person who is not familiar with something already to figure out whether they should be interested in it or not. Ideally it should be several sentences of English text in a case like this.

Reply:
Thanks... I'll update it to give a little more context, although if someone doesn't know what gamestreaming or Moonlight is they likely will have no interest in Sunshine.